server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreBaseIntegTestCase.java

```diff
@@ -151,9 +153,10 @@ protected Collection<Class<? extends Plugin>> nodePlugins() {
 
     @Override
     protected Settings nodeSettings(int nodeOrdinal) {
-        if (segmentRepoPath == null || translogRepoPath == null) {
+        if (segmentRepoPath == null || translogRepoPath == null || routingTableRepoPath == null) {
             segmentRepoPath = randomRepoPath().toAbsolutePath();
             translogRepoPath = randomRepoPath().toAbsolutePath();
+            routingTableRepoPath = randomRepoPath().toAbsolutePath();
```

Comment:
If routingTableRepoPath is not added by the test, lets not initialize it. We need to ensure existing tests run without routing table enabled as well.

Reply:
Addressed in revision.